### ISSUE TITLE
Image component: cropping doesn't give the expected result

### ISSUE THREAD

**Issue by meols** [equinor/energyvision#775]

**Comment by meols** [comment]
When cropping the image with square and setting a focus point, my photo doesn't show a cropped version of the image on my page. How can I make a 16:9 image of this example (and for any image)?
![image.png](https://images.zenhubusercontent.com/5fbbf4f2bab8ae744cb6560f/1755b074-3be3-43c1-83b2-4755c0b7fefd)

**Comment by wenche** [comment]
This one is actually related to #267 and #268. The Image component does not use hotspots, cropping and aspect ratios. The reason for this decision is that the image component needs to support a broader range of images beyond more decorative ones. Examples include maps, drawing and images that shouldn't be restricted like the map on [the Angola page
](https://www.equinor.com/en/where-we-are/angola.html). 

Since Sanity (at that point at least, we should check this and see if it's changed) doesn't support just cropping (#267)and not hotspot the component in Sanity still shows the cropping and hotspot features even if the web doesn't use it. (#268)

In other words, the image component will always show the full image. If you upload an image with aspect ratio like 5:25 it will take a lot of space :)

**Comment by nicholas-haggerty** [comment]
@wenche Not sure I understand that decision? And the two referenced cards don't really shed much light. My understanding of having a crop tool (and hotspot) meant we did have the option to leave the image 'as is' (full crop)  - to cover maps/drawings etc. But if I want to focus in and/or crop...... I can. This renders the functionality essentially pointless and means we have to go back to download/crop/upload an image to solve the scenario above. Which was one major point of switching CMS. And impacts on the GDPR restrictions/adding meta data for editors. What are the alternates here? Create a separate Fixed height/width image component to then allow for the crop/hotspot? (Like the top banner image)

**Comment by nicholas-haggerty** [comment]
Is this task here to make an options in the image component? 16/9 should be the default display (allowing for crop and hotspot) with an option to display the image based on its dimensions (for maps/illustrations) which wouldn't have the crop options (as it is today) Or would we need multiple image components to solve different requirements?

**Comment by nicholas-haggerty** [comment]
Related to images we potentially should consider an option to display a smaller image to solve pages like this (work in progress!) Where ideally the image of the cec member would be smaller than the proposed example where they would all display different sizes based on whats in BM. Even with a 16/9 component display as proposed above I think would be too large. https://equinor.sanity.studio/desk/topicContent;9eea162f-72ba-44c5-ba4c-86eee2b58cbc%2Cview%3Dpreview
Thoughts? @BirteThornquist

**Comment by BirteThornquist** [comment]
I can definitely see that the image component will not be optimal for the use case described here @nicholas-haggerty . For images in the news template editor we have the option for small or big image. Guess we could have a similar solution also for the image component for topics...

**Comment by fernandolucchesi** [comment]
Check with sanity if we can have a pre-defined crop area for different dimensions

**Comment by fernandolucchesi** [comment]
Probably fixed by Fotoware

**Comment by fernandolucchesi** [comment]
@meols, do we still need this after fotoware?
